make/Makefile.iso

```diff
@@ -14,6 +14,11 @@ ifneq ($(strip $(REPO_CACHE)),)
 	BUILD_ARGS+=--image-repository $(REPO_CACHE)
 endif
 
+MKSQUASHFS?=$(shell which mksquashfs 2> /dev/null)
+ifeq ("$(MKSQUASHFS)","")
+MKSQUASHFS="/usr/bin/mksquashfs"
```

Comment:
similar thing should be done for `xorriso` too :+1:

Reply:
Let me merge this one first and continue testing. I'm sure I'll find even more :wink: